packages/localizations/src/en-US.ts

```diff
@@ -52,7 +52,7 @@ export const enUS: LocalizationResource = {
   formFieldInputPlaceholder__organizationSlug: '',
   formFieldError__notMatchingPasswords: `Passwords don't match.`,
   formFieldError__matchingPasswords: 'Passwords match.',
-  formFieldAction__forgotPassword: 'Forgot password',
+  formFieldAction__forgotPassword: 'Forgot password?',
```

Comment:
🙃 can we add the question mark to the rest of the localization packages as well?